### ISSUE TITLE
Error: Can't resolve 'fs' in '....\node_modules\electron'

### ISSUE THREAD

**Issue by ting-dev-coder** [andotorg/electron-vue-app-template#2]
### Issue

> in ./node_modules/electron/index.js
 Module not found: Error: Can't resolve 'fs' in 'C:\Users\USER\Desktop\Ting\music-player\node_modules\electron'
in ./node_modules/electron/index.js
Module not found: Error: Can't resolve 'path' in 'C:\Users\USER\Desktop\Ting\music-player\node_modules\electron'
### Describe
when I  import { remote } from 'electron', it showed the error, how to fix it?
If I need to set some settings that should be set in webpack.config.js, which file I can use?

**Comment by andotorg** [comment]
import fs after show error?

**Comment by ting-dev-coder** [comment]
> import fs after show error?

no, when I  import { remote } from 'electron', it showed the error

**Comment by ting-dev-coder** [comment]
anyone can help?

**Comment by andotorg** [comment]
https://zhuanlan.zhihu.com/p/406184033

**Comment by andotorg** [comment]
replace method： https://www.jianshu.com/p/3dc8309533e7

**Comment by webbrain-one** [comment]
Hi there! 👋 Thanks for reporting this issue.

This error happens because Webpack is attempting to bundle Node.js modules like `fs` and `path`, which aren't available in the browser environment. I've opened [a PR to fix this](https://github.com/andotorg/electron-vue-app-template/pull/3) by adjusting the webpack configuration to handle these dependencies correctly.

Feel free to take a look and let me know if this resolves the problem for you! Thanks!
